bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/handler/SunSynkAccountHandler.java

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sunsynk.internal.handler;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.openhab.binding.sunsynk.internal.api.dto.APIdata;
+import org.openhab.binding.sunsynk.internal.api.dto.Client;
+import org.openhab.binding.sunsynk.internal.api.dto.Details;
+import org.openhab.binding.sunsynk.internal.classes.Inverter;
+import org.openhab.binding.sunsynk.internal.config.SunSynkAccountConfig;
+import org.openhab.core.io.net.http.HttpUtil;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * The {@link SunSynkAccountHandler} is responsible for handling the SunSynk Account Bridge
+ *
+ *
+ * @author Lee Charlton - Initial contribution
+ */
+
+// @NonNullByDefault
+public class SunSynkAccountHandler extends BaseBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(SunSynkAccountHandler.class);
+    private Client sunAccount = new Client();
+
+    public SunSynkAccountHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(@NonNull ChannelUID channelUID, @NonNull Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.ONLINE);
+        logger.debug("SunSynk Handler Intialised attempting to retrieve configuration");
+        configAccount();
```

Comment:
This should be run in a scheduled task / async as the init method should return quick

Reply:
Done